app/recipies/page.tsx

```diff
@@ -0,0 +1,59 @@
+import Link from "next/link";
+import { ReactElement, JSXElementConstructor, ReactNode, ReactPortal, PromiseLikeOfReactNode } from "react";
+
+type Product = {
+    id: number;
+    title: string;
+    description: string;
+    images: string[]
+}
+
+
+function CategoryButton({ item }: { item: { id: number; images: (string | undefined)[]; title: string | null | undefined; }; }) {
```

Comment:
Similarily to before I think these Props should be defined as a separate type above